puppet/zulip/files/nginx/sites-available/zulip-enterprise

```diff
@@ -24,6 +24,12 @@ server {
         alias /home/zulip/uploads/avatars;
     }
 
+    location /realms {
+        add_header X-Content-Type-Options nosniff;
+        include /etc/nginx/zulip-include/uploads.types;
+        alias /home/zulip/uploads/realms;
+    }
+
```

Comment:
We should use the same storage strategy as with avatars, and not need this directory or the corresponding config here or in `dev_urls.py`.